x-pack/test_serverless/api_integration/test_suites/common/alerting/helpers/alerting_api_helper.ts

```diff
@@ -28,18 +34,22 @@ interface CreateEsQueryRuleParams {
 }
 
 export async function createIndexConnector({
-  supertest,
+  supertestWithoutAuth,
+  roleAuthc,
+  internalReqHeader,
```

Comment:
Similar to my comment in `x-pack/test_serverless/api_integration/services/alerting_api.ts` - but here we are not in the context of a service.
We could leave this as is, but the proper solution would be to convert this helper file into an actual service (or include it in the alerting_api service). Could also leave it as a follow-up for the test owners.

Reply:
I'd like to follow this up, right after this pr.